.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ orbs:
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.6.3-stretch-node
+      - image: cimg/ruby:3.1.1
```

Comment:
The previous image was legacy as noted [here](https://circleci.com/developer/images/image/cimg/ruby):

> cimg/ruby
> This image is designed to supercede the legacy CircleCI Ruby image, circleci/ruby.